nntrainer/src/blas_interface.cpp

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0-only
+/**
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file	blas_interface.cpp
+ * @date	28 Aug 2020
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ * @brief	This is dummy header for blas support
+ *
+ */
+
+#include <blas_interface.h>
+
+namespace nntrainer {
+
+#ifndef USE_BLAS
+static void saxpy_raw(const unsigned int N, const float alpha, const float *X,
+                      const int incX, float *Y, const int incY) {
+  if (incX <= 0 or incY <= 0)
+    throw std::invalid_argument(
+      "Error: negative inc not supported without cblas");
+  for (unsigned int i = 0; i < N; ++i)
+    Y[i] = Y[i * incY] + X[i * incX] * alpha;
```

Comment:
I guess this should be according to the [reference](
https://www.ibm.com/support/knowledgecenter/SSFHY8_6.2/reference/am5gr_hsaxpy.html#am5gr_hsaxpy__am5gr_fsaxpy)

```suggestion
    Y[i * incY] = Y[i * incY] + X[i * incX] * alpha;
```

Reply:
fixed. Thanks.